### PR TITLE
Shortcut Sample Addin

### PR DESCRIPTION
**Pull request by nicheungMSFT**
Shortcut sample addin without custom functions

### CODE REVIEW

File: addins/ShortcutSample/ShortcutSample.xml [hunk 1]
```diff
@@ -0,0 +1,154 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--Created: c5a33478-ba81-414d-ad52-804f5da8facf -->
```
Comment (zlatko-michailov) [inline addins/ShortcutSample/ShortcutSample.xml:2]: The GUIDs don't match.

File: addins/ShortcutSample/ShortcutSample.xml [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+    xsi:type="TaskPaneApp">
+
+  <!-- Begin Basic Settings: Add-in metadata, used for all versions of Office unless override provided. -->
+
+  <!-- IMPORTANT! Id must be unique for your add-in, if you reuse this manifest ensure that you change this id to a new GUID. -->
+  <Id>d68dd406-771d-4dfc-ae80-8998b7821972</Id>
+
+  <!--Version. Updates from the store only get triggered if there is a version change. -->
+  <Version>1.0.0.0</Version>
+  <ProviderName>OEP Dev Team</ProviderName>
```
Comment (zlatko-michailov) [inline addins/ShortcutSample/ShortcutSample.xml:17]: _Microsoft_

File: addins/ShortcutSample/ShortcutSample.xml [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+  <DefaultLocale>en-US</DefaultLocale>
+
+  <!-- The display name of your add-in. Used on the store and various places of the Office UI such as the add-ins dialog. -->
+  <DisplayName DefaultValue="ShortcutSample" />
```
Comment (zlatko-michailov) [inline addins/ShortcutSample/ShortcutSample.xml:21]: This is what the user sees in the insertion dialog.

File: addins/ShortcutSample/ShortcutSample.xml [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+  <Description DefaultValue="ShortcutSample" />
```
Comment (zlatko-michailov) [inline addins/ShortcutSample/ShortcutSample.xml:22]: This is the tool tip when the user hovers over the icon in the insertion dialog.

File: addins/ShortcutSample/ShortcutSample.xml [hunk 5]
```diff
@@ -0,0 +1,154 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (zlatko-michailov) [inline addins/ShortcutSample/ShortcutSample.xml:1]: In general, I suggest cloning one of the samples we have here, and removing the parts that you don't need (as opposed to cloning an internal test add-in).

File: addins/ShortcutSample/ShortcutSample.xml [hunk 6]
```diff
@@ -0,0 +1,154 @@
...
+        <bt:String id="OEP.Group1Label" DefaultValue="Shortcut Sample" />
+        <bt:String id="OEP.GetStarted.Title" DefaultValue="Shortcut Sample" />
+      </bt:ShortStrings>
+      <!-- LongStrings max characters==250. -->
+      <bt:LongStrings>
+        <bt:String id="OEP.TaskpaneButton.Tooltip" DefaultValue="Click to Show a Taskpane" />
+        <bt:String id="OEP.GetStarted.Description" DefaultValue="Your sample add-in loaded succesfully. Go to the HOME tab and click the 'Show Taskpane' button to get started." />
+      </bt:LongStrings>
+    </Resources>
+    <WebApplicationInfo>
```
Comment (zlatko-michailov) [inline addins/ShortcutSample/ShortcutSample.xml:140]: Do we need SSO for this add-in?

File: addins/ShortcutSample/ShortcutSample.xml (hunk 5 above)
Comment (zlatko-michailov) [inline addins/ShortcutSample/ShortcutSample.xml:1]: I don't see a `README.md` in this PR. It is important. Look at the READMEs we have here, and pick one that fits your purpose.

File: addins/ShortcutSample/ShortcutSample.xml (hunk 4 above)
Comment (nicheungMSFT) [reply addins/ShortcutSample/ShortcutSample.xml:22]: I'm not sure what I need to change this to, I looked at CFSample2 they seem to only put the addin name here

File: addins/ShortcutSample/ShortcutSample.xml (hunk 3 above)
Comment (nicheungMSFT) [reply addins/ShortcutSample/ShortcutSample.xml:21]: I'm not sure what to change this to, in CFSample2 they put the addin name here

Comment (zlatko-michailov) [reply addins/ShortcutSample/ShortcutSample.xml:21]: At least make it two words. Short is fine, but it should look like a title.

File: addins/ShortcutSample/ShortcutSample.xml (hunk 4 above)
Comment (zlatko-michailov) [reply addins/ShortcutSample/ShortcutSample.xml:22]: Same as the above - at least split it into words.

File: addins/ShortcutSample/ShortcutSample.xml (hunk 3 above)
Comment (zlatko-michailov) [reply addins/ShortcutSample/ShortcutSample.xml:21]: You can also look at https://github.com/OfficeDev/custom-functions/tree/master/addins/upgrade. There are two manifests (xml files) there.

**Pull request merged**